package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^2.2.0-rc"
```

Comment:
Funny that you should mention it, but the reason we're on 2.4.1 at twitter is that  webpack/webpack#4815 introduced webpack/webpack#4850. There is a [workaround](https://github.com/webpack/webpack/issues/4795#issuecomment-302824613) for the issue that works in 2.4, but not 2.5 and above.

Regardless, every project that uses GWP with webpack is responsible for finding the right mix of versions that works for them; I don't know if GWP needs to have an opinion here outside of the specific things that it needs from webpack. GWP does not have anything in particular that depends on 2.5.1.